### PR TITLE
Update README-NL.md

### LINKED ISSUE #480: Help finish the Dutch translation of the README.md
The Dutch translation of the README is incomplete (https://github.com/freeCodeCamp/how-to-contribute-to-open-source/blob/master/README-NL.md) and we need help to finish it.

Your translations can be partial. If possible, please have another person who knows Dutch to review your pull request.

If you create a partial translation, feel free to simply refer to this issue as 

```
Related #480
```

in the pull request message body.

### PR DESCRIPTION
**Pull request by herkura**
Closed #480

### CODE REVIEW

**Comment by herkura** [comment]
@erictleung  pls see this issue
i have some problem in checks

**Comment by erictleung** [comment]
@herkura thanks for the translations! I don't have the expertise to review this.

@Lorretje are you able to review this translation? If not, no worries!

**Comment by erictleung** [comment]
@herkura because of some other pull requests that were merged for this README, there are some conflicts that can't be completed in GitHub's web interface. So you'll need to pull updates before going forward.

```
git clone https://github.com/herkura/how-to-contribute-to-open-source
cd how-to-contribute-to-open-source
git remote add upstream https://github.com/freeCodeCamp/how-to-contribute-to-open-source
git fetch upstream

git checkout update-README-NL
git rebase upstream/master
```

Then you can make your changes before pushing your changes back up to GitHub.

I hope that helps.

**Comment by herkura** [comment]
@erictleung Thanks for the update but is did the same process that you mentioned .. I will try again Thanks again

File: README-NL.md [hunk 1]
```diff
@@ -44,7 +44,27 @@ Als je vragen of opmerkingen hebt, kan je een issue aanmaken.
...
+- [45 Github-problemen: wel en niet](https://hackernoon.com/45-github-issues-dos-and-donts-dfec9ab4b612) -Do's en Don'ts op GitHub.
+- [GitHub-handleidingen](https://guides.github.com/) -basisgidsen voor het effectief gebruiken van GitHub.
+- [Draag bij aan Open Source](https://github.com/danthareja/contribute-to-open-source) - Leer de GitHub-workflow door code bij te dragen aan een simulatieproject.
+- [Linux Foundation's open source-handleidingen voor de onderneming](https://www.linuxfoundation.org/resources/open-source-guides/) - De handleidingen van de Linux Foundation voor Open Source-projecten.
+- [CSS-trucs Een open source-etiquettegids](https://css-tricks.com/open-source-etiquette-guidebook/) -Een Open Source Etiquette-handboek, geschreven door Kent C. Dodds en Sarah Drasner.
+- [A tot Z bronnen voor studenten](https://github.com/dipakkr/A-to-Z-Resources-for-Students) -Een Open Source Etiquette-handboek, geschreven door Kent C. Dodds en Sarah Drasner.An Open Source Etiquette Guidebook, geschreven door Kent C. Dodds en Sarah Drasner.
```
Comment (noah-vdv) [inline README-NL.md:58]:
```suggestion
- [A tot Z bronnen voor studenten](https://github.com/dipakkr/A-to-Z-Resources-for-Students) -Een Open Source Etiquette-handboek, geschreven door Kent C. Dodds en Sarah Drasner.
```

File: README-NL.md [hunk 2]
```diff
@@ -44,7 +44,27 @@ Als je vragen of opmerkingen hebt, kan je een issue aanmaken.
...
+- [Een inleiding tot Open Source](https://www.digitalocean.com/community/tutorial_series/an-introduction-to-open-source) - Tutorials van DigitalOcean om u te begeleiden op weg naar succes bij uw bijdragen hier op GitHub.
+- [Issuehub.io](http://issuehub.io/) - een tool voor het zoeken naar GitHub-problemen op label en taal.
+- [Code Triage](https://www.codetriage.com/) -een andere, erg leuke tool voor het vinden van populaire repositories en problemen gefilterd op taal.
+- [Awesome-for-beginners](https://github.com/MunGell/awesome-for-beginners) - een GitHub-repo die projecten verzamelt met goede bugs voor nieuwe bijdragers, en labels toepast om ze te beschrijven.
+- [Open source-handleidingen](https://opensource.guide/) - Verzameling van bronnen voor individuen, gemeenschappen en bedrijven die willen leren hoe ze een Open Source-project kunnen runnen en eraan kunnen bijdragen.
+- [45 Github-problemen: wel en niet](https://hackernoon.com/45-github-issues-dos-and-donts-dfec9ab4b612) -Do's en Don'ts op GitHub.
+- [GitHub-handleidingen](https://guides.github.com/) -basisgidsen voor het effectief gebruiken van GitHub.
```
Comment (noah-vdv) [inline README-NL.md:54]:
```suggestion
- [GitHub-handleidingen](https://guides.github.com/) - basisgidsen voor het effectief gebruiken van GitHub.
```

File: README-NL.md [hunk 3]
```diff
@@ -44,7 +44,27 @@ Als je vragen of opmerkingen hebt, kan je een issue aanmaken.
 - [Licenties](#license)
 
 ## Bijdragen aan een Open Source project in het algemeen
-
+- [De definitieve gids om bij te dragen aan open source](https://www.freecodecamp.org/news/the-definitive-guide-to-contributing-to-open-source-900d5f9f2282/) door [@DoomHammerNG](https://twitter.com/DoomHammerNG)
+- [Een inleiding tot Open Source](https://www.digitalocean.com/community/tutorial_series/an-introduction-to-open-source) - Tutorials van DigitalOcean om u te begeleiden op weg naar succes bij uw bijdragen hier op GitHub.
+- [Issuehub.io](http://issuehub.io/) - een tool voor het zoeken naar GitHub-problemen op label en taal.
+- [Code Triage](https://www.codetriage.com/) -een andere, erg leuke tool voor het vinden van populaire repositories en problemen gefilterd op taal.
```
Comment (noah-vdv) [inline README-NL.md:50]:
```suggestion
- [Code Triage](https://www.codetriage.com/) - een andere, erg leuke tool voor het vinden van populaire repositories en problemen gefilterd op taal.
```

File: README-NL.md [hunk 4]
```diff
@@ -44,7 +44,27 @@ Als je vragen of opmerkingen hebt, kan je een issue aanmaken.
...
+- [GitHub-handleidingen](https://guides.github.com/) -basisgidsen voor het effectief gebruiken van GitHub.
+- [Draag bij aan Open Source](https://github.com/danthareja/contribute-to-open-source) - Leer de GitHub-workflow door code bij te dragen aan een simulatieproject.
+- [Linux Foundation's open source-handleidingen voor de onderneming](https://www.linuxfoundation.org/resources/open-source-guides/) - De handleidingen van de Linux Foundation voor Open Source-projecten.
+- [CSS-trucs Een open source-etiquettegids](https://css-tricks.com/open-source-etiquette-guidebook/) -Een Open Source Etiquette-handboek, geschreven door Kent C. Dodds en Sarah Drasner.
+- [A tot Z bronnen voor studenten](https://github.com/dipakkr/A-to-Z-Resources-for-Students) -Een Open Source Etiquette-handboek, geschreven door Kent C. Dodds en Sarah Drasner.An Open Source Etiquette Guidebook, geschreven door Kent C. Dodds en Sarah Drasner.
+- [Pull Request Roulette](http://www.pullrequestroulette.com/) - Deze site heeft een lijst met pull-aanvragen die ter beoordeling zijn ingediend en behoren tot Open Source-projecten die op Github worden gehost.
```
Comment (noah-vdv) [inline README-NL.md:59]:
```suggestion
- [Pull Request Roulette](http://www.pullrequestroulette.com/) - Deze site heeft een lijst met pull requests die ter beoordeling zijn ingediend en behoren tot Open Source-projecten die op Github worden gehost.
```

File: README-NL.md [hunk 5]
```diff
@@ -44,7 +44,27 @@ Als je vragen of opmerkingen hebt, kan je een issue aanmaken.
...
+- ["Hoe u kunt bijdragen aan een open source-project op GitHub" door Egghead.io](https://egghead.io/courses/how-to-contribute-to-an-open-source-project-on-github) - Een stapsgewijze videogids over hoe u kunt beginnen met bijdragen aan Open Source-projecten op GitHub.
+- [Bijdragen aan open source: een live walkthrough van begin tot eind](https://medium.com/@kevinjin/contributing-to-open-source-walkthrough-part-0-b3dc43e6b720) - Deze walkthrough van een open source-bijdrage omvat alles, van het kiezen van een geschikt project, het werken aan een probleem, tot het samenvoegen van de PR.
```
Comment (noah-vdv) [inline README-NL.md:61]:
```suggestion
- [Bijdragen aan open source: een live walkthrough van begin tot eind](https://medium.com/@kevinjin/contributing-to-open-source-walkthrough-part-0-b3dc43e6b720) - Deze walkthrough van een open source-bijdrage omvat alles, van het kiezen van een geschikt project, het werken aan een probleem (issue), tot het samenvoegen van de PR.
```

File: README-NL.md [hunk 6]
```diff
@@ -44,7 +44,27 @@ Als je vragen of opmerkingen hebt, kan je een issue aanmaken.
...
+- ["Hoe u kunt bijdragen aan een open source-project door" Sarah Drasner](https://css-tricks.com/how-to-contribute-to-an-open-source-project/) - Ze concentreren zich op de essentie van het bijdragen van een pull request (PR) aan het project van iemand anders op GitHub.
```
Comment (noah-vdv) [inline README-NL.md:62]:
```suggestion
- ["Hoe u kunt bijdragen aan een open source-project" door Sarah Drasner](https://css-tricks.com/how-to-contribute-to-an-open-source-project/) - Ze concentreren zich op de essentie van het bijdragen van een pull request (PR) aan het project van iemand anders op GitHub.
```

File: README-NL.md [hunk 7]
```diff
@@ -44,7 +44,27 @@ Als je vragen of opmerkingen hebt, kan je een issue aanmaken.
...
+- ["Aan de slag met Open Source door" Sayan Chowdhury](https://www.hackerearth.com:443/getstarted-opensource/) -Dit artikel behandelt de bronnen om bij te dragen aan open source voor beginners op basis van hun favoriete taalinteresse.
```
Comment (noah-vdv) [inline README-NL.md:63]:
```suggestion
- ["Aan de slag met Open Source" door Sayan Chowdhury](https://www.hackerearth.com:443/getstarted-opensource/) - Dit artikel behandelt de bronnen om bij te dragen aan open source voor beginners op basis van hun favoriete taalinteresse.
```

File: README-NL.md [hunk 8]
```diff
@@ -44,7 +44,27 @@ Als je vragen of opmerkingen hebt, kan je een issue aanmaken.
...
+- ["Blader door goede eerste nummers om bij te dragen aan open source"](https://github.blog/2020-01-22-browse-good-first-issues-to-start-contributing-to-open-source/) -GitHub helpt je nu bij het vinden van goede eerste problemen om bij te dragen aan open source.
```
Comment (noah-vdv) [inline README-NL.md:64]:
```suggestion
- ["Blader door good first issues om bij te dragen aan open source"](https://github.blog/2020-01-22-browse-good-first-issues-to-start-contributing-to-open-source/) - GitHub helpt je nu bij het vinden van good first issues om bij te dragen aan open source.
```

File: README-NL.md [hunk 9]
```diff
@@ -58,15 +78,47 @@ Zoek links die rechtstreeks verwijzen naar geschikte problemen om aan bij te dra
...
+- [GitHub Learning Resources](https://docs.github.com/en/github/getting-started-with-github/git-and-github-learning-resources) -Leermiddelen voor Git en GitHub.
+- [Pro Git](https://git-scm.com/book/en/v2) - Het volledige Pro Git-boek, geschreven door Scott Chacon en Ben Straub en uitgegeven door Apress.
+- [Git-it](https://github.com/jlord/git-it-electron) - Stap voor stap Git tutorial desktop-app.
+- [Vliegregels voor Git](https://github.com/k88hudson/git-flight-rules) - Een gids over wat u moet doen als er iets misgaat.
+- [Git-gids voor beginners in het Spaans](https://platzi.github.io/git-slides/#/) - Een complete gids met dia's over git en GitHub uitgelegd in het Spaans. Een volledige guía de diapositivas sobre git y GitHub explicadas en Español.
+- [Git Kraken](https://www.gitkraken.com/git-client) - Visueel, platformonafhankelijk en interactief `git` desktop applicatie voor versiebeheer.
+- [Git-tips](https://github.com/git-tips/tips) -Verzameling van de meest gebruikte git tips en trucs.
+- [Git Best Practices](https://sethrobertson.github.io/GitBestPractices/) -Vaak vastleggen, later perfectioneren, een keer publiceren: Git Best Practices.
```
Comment (noah-vdv) [inline README-NL.md:105]:
```suggestion
- [Denk als (een) Git](http://think-like-a-git.net/) - Git introductie voor "gevorderde beginners", maar worstelen nog steeds, om je een eenvoudige strategie te geven om veilig met git te experimenteren.
- [Probeer Git](https://try.github.io/) - Leer Git gratis in 15 minuten vanuit je browser.
- [Elke dag Git](https://git-scm.com/docs/giteveryday) - Een nuttige minimale set commando's voor elke dag gebruik van Git.
- [Oh shit, git!](https://ohshitgit.com/) - hoe je uit veelvoorkomende `git`-fouten kunt komen die in gewoon Engels worden beschreven; zie ook [Dangit, git!](https://dangitgit.com/) voor de pagina zonder vloeken.
- [Atlassian Git Tutorials](https://www.atlassian.com/git/tutorials) - verschillende tutorials over het gebruik van`git`.
- [GitHub Git Cheat Sheet](https://education.github.com/git-cheat-sheet-education.pdf) (PDF)
- [freeCodeCamp's Wiki op Git-bronnen](https://forum.freecodecamp.org/t/wiki-git-resources/13136)
- [GitHub Flow](https://www.youtube.com/watch?v=juLIxo42A_s) (42:06) - GitHub gesprek over het maken van een pull request.
- [GitHub Learning Resources](https://docs.github.com/en/github/getting-started-with-github/git-and-github-learning-resources) - Leermiddelen voor Git en GitHub.
- [Pro Git](https://git-scm.com/book/en/v2) - Het volledige Pro Git-boek, geschreven door Scott Chacon en Ben Straub en uitgegeven door Apress.
- [Git-it](https://github.com/jlord/git-it-electron) - Stap voor stap Git tutorial desktop-app.
- [Vliegregels voor Git](https://github.com/k88hudson/git-flight-rules) - Een gids over wat u moet doen als er iets misgaat.
- [Git-gids voor beginners in het Spaans](https://platzi.github.io/git-slides/#/) - Een complete gids met dia's over git en GitHub uitgelegd in het Spaans.
- [Git Kraken](https://www.gitkraken.com/git-client) - Visueel, platformonafhankelijk en interactief `git` desktop applicatie voor versiebeheer.
- [Git-tips](https://github.com/git-tips/tips) - Verzameling van de meest gebruikte git tips en trucs.
- [Git Best Practices](https://sethrobertson.github.io/GitBestPractices/) - Vaak vastleggen, later perfectioneren, een keer publiceren: Git Best Practices.
```

File: README-NL.md [hunk 10]
```diff
@@ -58,15 +78,47 @@ Zoek links die rechtstreeks verwijzen naar geschikte problemen om aan bij te dra
...
+- [Open Sources 2.0: de voortdurende evolutie](https://archive.org/details/opensources2.000diborich) (2005) - Open Sources 2.0 is een verzameling inzichtelijke en tot nadenken stemmende essays van hedendaagse technologieleiders die het evolutionaire beeld blijven schetsen dat zich ontwikkelde in het boek uit 1999, Open Sources: Voices from the Revolution.
+- [De architectuur van open source-applicaties](http://www.aosabook.org/en/git.html) - Laat zien hoe verschillende aspecten van Git onder de dekens werken om gedistribueerde workflows mogelijk te maken, en hoe het verschilt van andere versiebeheersystemen (VCS'en).
+- [Open bronnen: stemmen uit de open source-revolutie](https://www.oreilly.com/openbook/opensources/book/) -Essays van open-sourcepioniers zoals Linus Torvalds (Linux), Larry Wall (Perl) en Richard Stallman (GNU). ## Open Source bijdrage-initiatieven.
```
Comment (noah-vdv) [inline README-NL.md:114]:
```suggestion
- [Open Sources: stemmen uit de open source-revolutie](https://www.oreilly.com/openbook/opensources/book/) - Essays van open-sourcepioniers zoals Linus Torvalds (Linux), Larry Wall (Perl) en Richard Stallman (GNU).
```

File: README-NL.md [hunk 11]
```diff
@@ -58,15 +78,47 @@ Zoek links die rechtstreeks verwijzen naar geschikte problemen om aan bij te dra
...
+- [Alleen eerste timers](https://www.firsttimersonly.com/) -Een lijst met bugs die zijn gelabeld "alleen first-timers".
+- [Eerste bijdragen](https://firstcontributions.github.io/) - Lever uw eerste Open Source-bijdrage in 5 minuten. Een tool en tutorial om beginners op weg te helpen met bijdragen. [Here](https://github.com/firstcontributions/first-contributions) is de GitHub-broncode voor de site en de mogelijkheid om een ​​bijdrage te leveren aan de repository zelf.
+- [Hacktoberfest](https://hacktoberfest.digitalocean.com/) -Een programma om Open Source-bijdragen aan te moedigen. Verdien cadeaus zoals t-shirts en stickers door ten minste vier trekverzoeken in de maand oktober te doen.
+- [24 Pull Requests](https://24pullrequests.com) - 24 Pull Requests is een project om Open Source samenwerking in de maand december te promoten.
+- [Ovio](https://ovio.org) -Een platform met een samengestelde selectie van bijdragervriendelijke projecten. Het heeft een [krachtige tool voor het zoeken naar problemen] (https://ovio.org/issues) waarmee u projecten en problemen kunt opslaan voor later.
```
Comment (noah-vdv) [inline README-NL.md:121]:
```suggestion
- [Voor het oprapen](https://up-for-grabs.net/) - Bevat projecten met beginnersvriendelijke problemen.
- [First timers only](https://www.firsttimersonly.com/) - Een lijst met bugs die zijn gelabeld "first timers only".
- [Eerste bijdragen](https://firstcontributions.github.io/) - Lever uw eerste Open Source-bijdrage in 5 minuten. Een tool en tutorial om beginners op weg te helpen met bijdragen. [Hier](https://github.com/firstcontributions/first-contributions) is de GitHub-broncode voor de site en de mogelijkheid om een ​​bijdrage te leveren aan de repository zelf.
- [Hacktoberfest](https://hacktoberfest.digitalocean.com/) - Een programma om Open Source-bijdragen aan te moedigen. Verdien cadeaus zoals t-shirts en stickers door ten minste vier pull requests in de maand oktober te doen.
- [24 Pull Requests](https://24pullrequests.com) - 24 Pull Requests is een project om Open Source samenwerking in de maand december te promoten.
- [Ovio](https://ovio.org) - Een platform met een samengestelde selectie van bijdragervriendelijke projecten. Het heeft een [krachtige tool voor het zoeken naar issues](https://ovio.org/issues) waarmee u projecten en problemen kunt opslaan voor later.
```

**Review by noah-vdv — CHANGES_REQUESTED** [review]
Mostly a couple of nitpicks.
Also, probably a question for @erictleung : Should titles of articles, sites, etc. be translated or not?

File: README-NL.md [hunk 12]
```diff
@@ -58,15 +78,47 @@ Zoek links die rechtstreeks verwijzen naar geschikte problemen om aan bij te dra
 - [is:issue is:open label:up-for-grabs](https://github.com/search?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Aup-for-grabs)
 
 ## Het bijdragende ecosysteem van Mozilla**
```
Comment (noah-vdv) [inline README-NL.md:80]:
```suggestion
## Het bijdragende ecosysteem van Mozilla
```

File: README-NL.md [hunk 13]
```diff
@@ -58,15 +78,47 @@ Zoek links die rechtstreeks verwijzen naar geschikte problemen om aan bij te dra
...
+- [Goede eerste bugs](https://bugzil.la/sw:%22[good%20first%20bug]%22&limit=0) - bugs that developers have identified as a good introduction to the project.
+- [Mentored Bugs](https://bugzilla.mozilla.org/buglist.cgi?quicksearch=mentor%3A%40) - bugs that have a mentor assigned who will be there on IRC to help you when you get stuck while working on a fix.
+- [Bugs Ahoy](https://www.joshmatthews.net/bugsahoy/) - a site dedicated to finding bugs on Bugzilla.
+- [Firefox DevTools](http://firefox-dev.tools/) - a site dedicated to bugs filed for the developer tools in the Firefox browser.
+- [Wat kan ik doen voor Mozilla](https://whatcanidoformozilla.org/) - figure out what you can work on by answering a bunch of questions about your skill set and interests.
+- [Start Mozilla](https://twitter.com/StartMozilla) - a Twitter account that tweets about issues fit for contributors new to the Mozilla ecosystem.
 
 ## Versie control gebruiken**
```
Comment (noah-vdv) [inline README-NL.md:88]:
```suggestion
## Versie control gebruiken
```

File: README-NL.md [hunk 14]
```diff
@@ -58,15 +78,47 @@ Zoek links die rechtstreeks verwijzen naar geschikte problemen om aan bij te dra
...
+- [Pro Git](https://git-scm.com/book/en/v2) - Het volledige Pro Git-boek, geschreven door Scott Chacon en Ben Straub en uitgegeven door Apress.
+- [Git-it](https://github.com/jlord/git-it-electron) - Stap voor stap Git tutorial desktop-app.
+- [Vliegregels voor Git](https://github.com/k88hudson/git-flight-rules) - Een gids over wat u moet doen als er iets misgaat.
+- [Git-gids voor beginners in het Spaans](https://platzi.github.io/git-slides/#/) - Een complete gids met dia's over git en GitHub uitgelegd in het Spaans. Een volledige guía de diapositivas sobre git y GitHub explicadas en Español.
+- [Git Kraken](https://www.gitkraken.com/git-client) - Visueel, platformonafhankelijk en interactief `git` desktop applicatie voor versiebeheer.
+- [Git-tips](https://github.com/git-tips/tips) -Verzameling van de meest gebruikte git tips en trucs.
+- [Git Best Practices](https://sethrobertson.github.io/GitBestPractices/) -Vaak vastleggen, later perfectioneren, een keer publiceren: Git Best Practices.
+- [Git interactieve zelfstudie](https://learngitbranching.js.org/) - Leer Git op de meest visuele en interactieve manier.
 
 ## Open Source boeken**
```
Comment (noah-vdv) [inline README-NL.md:108]:
```suggestion
## Open Source boeken
```

File: README-NL.md [hunk 15]
```diff
@@ -58,15 +78,47 @@ Zoek links die rechtstreeks verwijzen naar geschikte problemen om aan bij te dra
...
+- [Open Sources 2.0: de voortdurende evolutie](https://archive.org/details/opensources2.000diborich) (2005) - Open Sources 2.0 is een verzameling inzichtelijke en tot nadenken stemmende essays van hedendaagse technologieleiders die het evolutionaire beeld blijven schetsen dat zich ontwikkelde in het boek uit 1999, Open Sources: Voices from the Revolution.
+- [De architectuur van open source-applicaties](http://www.aosabook.org/en/git.html) - Laat zien hoe verschillende aspecten van Git onder de dekens werken om gedistribueerde workflows mogelijk te maken, en hoe het verschilt van andere versiebeheersystemen (VCS'en).
+- [Open bronnen: stemmen uit de open source-revolutie](https://www.oreilly.com/openbook/opensources/book/) -Essays van open-sourcepioniers zoals Linus Torvalds (Linux), Larry Wall (Perl) en Richard Stallman (GNU). ## Open Source bijdrage-initiatieven.
+## Open Source bijdrage-initiatieven**
```
Comment (noah-vdv) [inline README-NL.md:115]:
```suggestion
## Open Source bijdrage-initiatieven
```

**Review by noah-vdv — COMMENTED** [review]
Small fix

File: README-NL.md [hunk 16]
```diff
@@ -58,15 +78,47 @@ Zoek links die rechtstreeks verwijzen naar geschikte problemen om aan bij te dra
 - [is:issue is:open label:up-for-grabs](https://github.com/search?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Aup-for-grabs)
 
 ## Het bijdragende ecosysteem van Mozilla**
+- [Goede eerste bugs](https://bugzil.la/sw:%22[good%20first%20bug]%22&limit=0) - bugs that developers have identified as a good introduction to the project.
+- [Mentored Bugs](https://bugzilla.mozilla.org/buglist.cgi?quicksearch=mentor%3A%40) - bugs that have a mentor assigned who will be there on IRC to help you when you get stuck while working on a fix.
+- [Bugs Ahoy](https://www.joshmatthews.net/bugsahoy/) - a site dedicated to finding bugs on Bugzilla.
+- [Firefox DevTools](http://firefox-dev.tools/) - a site dedicated to bugs filed for the developer tools in the Firefox browser.
+- [Wat kan ik doen voor Mozilla](https://whatcanidoformozilla.org/) - figure out what you can work on by answering a bunch of questions about your skill set and interests.
+- [Start Mozilla](https://twitter.com/StartMozilla) - a Twitter account that tweets about issues fit for contributors new to the Mozilla ecosystem.
```
Comment (noah-vdv) [inline README-NL.md:86]:
```suggestion
- [Good first bug](https://bugzil.la/sw:%22[good%20first%20bug]%22&limit=0) - bugs die ontwikkelaars hebben geïdentificeerd als een goede introductie tot het project.
- [Mentored Bugs](https://bugzilla.mozilla.org/buglist.cgi?quicksearch=mentor%3A%40) - bugs waaraan een mentor is toegewezen die aanwezig zal zijn op IRC om je te helpen als je vastloopt tijdens het werken aan een oplossing.
- [Bugs Ahoy](https://www.joshmatthews.net/bugsahoy/) -een site gewijd aan het vinden van bugs op Bugzilla.
- [Firefox DevTools](http://firefox-dev.tools/) - een site gewijd aan bugs die zijn ingediend voor de ontwikkelaarstools in de Firefox-browser.
- [Wat kan ik doen voor Mozilla](https://whatcanidoformozilla.org/) - Zoek uit waar je aan kunt werken door een aantal vragen over je vaardigheden en interesses te beantwoorden.
- [Start Mozilla](https://twitter.com/StartMozilla) - een Twitter-account dat tweets plaatst over kwesties die geschikt zijn voor bijdragers die nieuw zijn in het Mozilla-ecosysteem.
```

**Comment by erictleung** [comment]
> 
> 
> Mostly a couple of nitpicks.
> Also, probably a question for @erictleung : Should titles of articles, sites, etc. be translated or not?

I don't mind either. But for simplicity, we can leave them alone for now unless someone wants to translate those as well.

**Review by Lorretje — APPROVED** [review]

**Pull request closed without merging**